parachain-template/node/src/service.rs

```diff
@@ -178,7 +180,18 @@ async fn build_relay_chain_interface(
 	match collator_options.relay_chain_rpc_url {
 		Some(relay_chain_url) => {
 			let client = create_client_and_start_worker(relay_chain_url, task_manager).await?;
-			Ok((Arc::new(RelayChainRpcInterface::new(client)) as Arc<_>, None))
+			let collator_pair = CollatorPair::generate().0;
+			let collator_node = cumulus_relay_chain_minimal_node::new_minimal_relay_chain(
+				polkadot_config,
+				collator_pair.clone(),
+				Arc::new(BlockChainRpcClient::new(client.clone())),
+			)
+			.await?;
+			task_manager.add_child(collator_node.task_manager);
+			Ok((
+				Arc::new(RelayChainRpcInterface::new(client, collator_node.overseer_handle)),
+				Some(collator_pair),
+			))
```

Comment:
Probably worth to put all of this into an internal function as we have it for the in process relay chain?